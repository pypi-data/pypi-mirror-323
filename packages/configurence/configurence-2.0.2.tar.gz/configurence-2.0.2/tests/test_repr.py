# -*- coding: utf-8 -*-


def test_repr(config, snapshot) -> None:
    assert repr(config) == snapshot
