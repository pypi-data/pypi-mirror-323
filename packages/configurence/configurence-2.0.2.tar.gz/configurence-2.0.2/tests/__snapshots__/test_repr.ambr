# serializer version: 1
# name: test_repr
  '''
  file: /Users/josh/.config/test-app.yaml
  name: test-app
  opt_bool: null
  opt_float: null
  opt_int: null
  opt_str: null
  some_bool: true
  some_float: 1.0
  some_int: 1
  some_other: default
  some_str: some_str
  
  '''
# ---
